src/lib/djerba/plugins/pwgs/case_overview/plugin.py

```diff
@@ -40,24 +40,25 @@ def configure(self, config):
     def extract(self, config):
         wrapper = self.get_config_wrapper(config)
         data = self.get_starting_plugin_data(wrapper, self.PLUGIN_VERSION)
-        results =  {
-                pc.ASSAY: "plasma Whole Genome Sequencing (pWGS) - 30X (v1.0)",
-                pc.PWGS_REPORT: config['core']['report_id'],
-                pc.PRIMARY_CANCER: config[self.identifier][pc.PRIMARY_CANCER],
-                pc.REQ_APPROVED: config[self.identifier][pc.REQ_APPROVED],
-                pc.DONOR: config[self.identifier][pc.DONOR],
-                pc.GROUP_ID: config[self.identifier][pc.GROUP_ID],
-                pc.PATIENT_ID: config[self.identifier][pc.PATIENT_ID_LOWER],
-                pc.STUDY:  config[self.identifier][pc.STUDY],
-                pc.WGS_REPORT: config[self.identifier][pc.WGS_REPORT]
-            }
+        results = {
+            pc.ASSAY: "plasma Whole Genome Sequencing (pWGS) - 30X (v1.0)",
+            pc.PWGS_REPORT: config['core']['report_id'],
+            pc.PRIMARY_CANCER: config[self.identifier][pc.PRIMARY_CANCER],
+            pc.REQ_APPROVED: config[self.identifier][pc.REQ_APPROVED],
+            pc.DONOR: config[self.identifier][pc.DONOR],
+            pc.GROUP_ID: config[self.identifier][pc.GROUP_ID],
+            pc.PATIENT_ID: config[self.identifier][pc.PATIENT_ID_LOWER],
+            pc.STUDY: config[self.identifier][pc.STUDY],
+            pc.WGS_REPORT: config[self.identifier][pc.WGS_REPORT]
+        }
         data[pc.RESULTS] = results
+        self.write_results_to_json(data)
```

Comment:
Instead of writing a new method, you can use `self.workspace.write_json`: https://github.com/oicr-gsi/djerba/blob/c7cc9d29e8a75456c1a4e0a3010883f0ae16c1fd/src/lib/djerba/core/workspace.py#L78